### PR TITLE
Update sqldb-or-3rdparty.md

### PR DESCRIPTION
**Pull request by igorpag**
Added more items related to Backup, Hybrid, Cross-DB access.

### CODE REVIEW

File: sql/sqldb-or-3rdparty.md [hunk 1]
```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
 
   Business Continuity Overview
   https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
+
+Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
+
+
+**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
```
Comment (dmakogon) [inline sql/sqldb-or-3rdparty.md]: If you wouldn't mind changing  "on-premise" to "on-premises" that would be great. It's in at least two places. Just want to make sure we get that right. :)

File: sql/sqldb-or-3rdparty.md [hunk 2]
```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
...
+| VERSIONING         | No control on upgrades                  | Full control over DB upgrade                 |
+| TCO                | Very low, self-managed                  | High (as on-premise)                         |
+| ADMINISTRATION     | No full DBA required                    | Full staffed DBA required                    |
+| MANAGEMENT         | Easy to manage many DBs                 | Complex to manage many DBs                   |
+| SCALE OUT          | Tools& Frameworks available             | No easy scale-out                            |
+| CONFIGURATION      | No setup customization                  | Full access to OS and SQL                    |
+| AUTHENTICATION     | Only SQL standard auth.                 | SQL standard and integrated                  |
+| SECURITY           | No Fixed IP fixed 1433 port             | Fixed IP, port can be change                 |
+| BACKUP             | Files not accessible, 35 days PITR      | Full control of backup files, unlimited PITR |
+| HYBRID             | No AlwaysOn AG support                  | Can join on-premise AlwaysOn AG topology     |
```
Comment (dmakogon) [inline sql/sqldb-or-3rdparty.md]: premise -> premises

File: sql/sqldb-or-3rdparty.md [hunk 3]
```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
...
+
+
+**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
+
+If you have existing SQL Server on-premise installation using AlwaysOn Availability Groups (AG) feature for high-availability and cross-datacenter disaster-recovery, a SQL Server instance installed inside Azure Virtual Machines can join this existing topology as an additional replica. This is not possible for Azure SQLDB. Additionally, if you want to use SQL Server “Log Shipping” DR feature, same restrictions apply.
+
+  High Availability and Disaster Recovery for SQL Server in Azure Virtual Machines
+  https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-sql-server-high-availability-and-disaster-recovery-solutions
+
+NOTE: Although Azure SQLDB can participate as a “Subscriber”, in a “Transactional Replication” topology with on-premise SQL Server instances or Virtual Machines in the Cloud, this is not considered HA/DR mechanism.
```
Comment (dmakogon) [inline sql/sqldb-or-3rdparty.md]: premise -> premises

File: sql/sqldb-or-3rdparty.md [hunk 4]
```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
 
   Business Continuity Overview
   https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
+
+Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
+
+
+**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
+
+If you have existing SQL Server on-premise installation using AlwaysOn Availability Groups (AG) feature for high-availability and cross-datacenter disaster-recovery, a SQL Server instance installed inside Azure Virtual Machines can join this existing topology as an additional replica. This is not possible for Azure SQLDB. Additionally, if you want to use SQL Server “Log Shipping” DR feature, same restrictions apply.
```
Comment (dmakogon) [inline sql/sqldb-or-3rdparty.md]: premise -> premises

Comment (igorpag) [inline sql/sqldb-or-3rdparty.md]: I will fix once back from vacation, thanks for catching it.
Regards.

---

Da: David Makogonmailto:notifications@github.com
Inviato: ‎24/‎12/‎2015 16:46
A: dmakogon/dbguidancemailto:dbguidance@noreply.github.com
Cc: Igor Pagliaimailto:igorpag@microsoft.com
Oggetto: Re: [dbguidance] Update sqldb-or-3rdparty.md (#28)

In sql/sqldb-or-3rdparty.mdhttps://github.com/dmakogon/dbguidance/pull/28#discussion_r48419108:

> @@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
> 
>    Business Continuity Overview
>    https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
> +
> +Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
> +
> +
> +**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
> +
> +If you have existing SQL Server on-premise installation using AlwaysOn Availability Groups (AG) feature for high-availability and cross-datacenter disaster-recovery, a SQL Server instance installed inside Azure Virtual Machines can join this existing topology as an additional replica. This is not possible for Azure SQLDB. Additionally, if you want to use SQL Server “Log Shipping” DR feature, same restrictions apply.

premise -> premises

**Comment by igorpag** [comment]
Fixed all "on-premise" and merged the branch.
thanks.

**Pull request merged**